src/CMakeLists.txt

```diff
@@ -12,3 +12,4 @@ set(SRC_FILES
     main.cpp)
 
 add_executable(main ${SRC_FILES})
+target_link_libraries(main m)
```

Comment:
Could you explain what this ```m``` is?

Reply:
Math dll libm. It was required for clang linkage, I believe.